recipe/meta.yaml

```diff
@@ -69,40 +89,42 @@ outputs:
 
         Supported Python Versions
         -------------------------
-        Python >= 3.6
+        Python >= 3.8
 
         Deprecated Python Versions
         --------------------------
-        Python == 2.7, Python == 3.5.
+        Python == 2.7, Python <= 3.7.
 
         The last version of this library compatible with Python 2.7
         and 3.5 is google-cloud-bigquery-storage==1.1.0.
-      doc_url: https://googleapis.dev/python/bigquerystorage/latest/index.html
+      doc_url: https://cloud.google.com/python/docs/reference/bigquerystorage/latest/summary_overview
       dev_url: https://github.com/googleapis/python-bigquery-storage
 
   - name: {{ name }}-core
     build:
-      noarch: python
-      script: {{ PYTHON }} -m pip install . -vv
+      skip: true  # [py<38]
```

Comment:
```suggestion
      # we skip py<38 following along conda-forge, to avoid importlib-metadata
      # which is needed for py37
      skip: true  # [py<38]
```